.stickler.yml

```diff
@@ -2,7 +2,7 @@ linters:
   flake8:
     max-line-length: 79
     fixer: false
-    ignore: I002
+    ignore: I002, F401, F403
```

Comment:
A better way to do this is to add `  # noqa:F401` (preceded by two spaces) at the end of the offending `import` line. That way, *actually* erroneous F401/F403 that other (less skilled) developers introduce will still be caught.

For F403, however, as a general preference I think it is better to explicitly import a long list of names:
```
from foo.bar import (
    apple,
    banana,
    carrot,
    # ...
)
```
…instead of `from foo.bar import *  # noqa:F403`. Two reasons:

1. In a top-level module (`__init__.py`) this makes it clear what's being presented to the user.
2. In a very long file, other developers will know that, if they change the behaviour of e.g. `carrot()`, they must look through the file to check whether it impacts other code.

Reply:
done